easybuild/easyconfigs/u/UCX/UCX-1.12.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.12.1'
+
+homepage = 'https://www.openucx.org/'
+description = """Unified Communication X
+An open-source production grade communication framework for data centric
+and high-performance applications
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/openucx/ucx/releases/download/v%(version)s']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+patches = [
+    'UCX-1.10.0-dynamic_modules.patch',
+    'UCX-1.11.2-missing_header.patch',
+]
+checksums = [
+    '40b447c8e7da94a253f2828001b2d76021eb4ad39647107d433d62d61e18ae8e',  # ucx-1.12.1.tar.gz
+    '3098c91f2aadc5f148466a2e8756769ad5e99f9515d8cfb6ad8a5712e7433408',  # UCX-1.10.0-dynamic_modules.patch
+    'fa9052a33da5c8e94bc7df0e5fe6ce3baf613310842ae1ab9ed4ef2d49666c01',  # UCX-1.11.2-missing_header.patch
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('Autotools', '20220317'),
+    ('pkgconf', '1.8.0'),
+]
+
+osdependencies = [OS_PKG_IBVERBS_DEV]
+
+dependencies = [
+    ('zlib', '1.2.12'),
+    ('numactl', '2.0.14'),
+]
+
+configure_cmd = "contrib/configure-release"
+
+configopts = '--enable-optimizations --enable-cma --enable-mt --with-verbs '
+configopts += '--without-java --disable-doxygen-doc '
```

Comment:
```suggestion
configopts += '--without-java --without-go --disable-doxygen-doc '
```

I'm seeing an error of
```
Making all in bindings/go
make[2]: Entering directory '/dev/shm/build-branfosj-admin/branfosj-admin-up/UCX/1.12.1/GCCcore-11.3.0/ucx-1.12.1/bindings/go'
mkdir -p /dev/shm/build-branfosj-admin/branfosj-admin-up/UCX/1.12.1/GCCcore-11.3.0/ucx-1.12.1/bindings/go/.libs/tmp
cd /dev/shm/build-branfosj-admin/branfosj-admin-up/UCX/1.12.1/GCCcore-11.3.0/ucx-1.12.1/bindings/go/src/ucx ;\
go build
/bin/sh: line 0: cd: /dev/shm/build-branfosj-admin/branfosj-admin-up/UCX/1.12.1/GCCcore-11.3.0/ucx-1.12.1/bindings/go/src/ucx: No such file or directory
go: go.mod file not found in current directory or any parent directory; see 'go help modules'
make[2]: *** [Makefile:598: build] Error 1
make[2]: Leaving directory '/dev/shm/build-branfosj-admin/branfosj-admin-up/UCX/1.12.1/GCCcore-11.3.0/ucx-1.12.1/bindings/go'
make[1]: *** [Makefile:743: all-recursive] Error 1
make[1]: Leaving directory '/dev/shm/build-branfosj-admin/branfosj-admin-up/UCX/1.12.1/GCCcore-11.3.0/ucx-1.12.1'
make: *** [Makefile:610: all] Error 2
```

If I disable the Go bindings then UCX builds. I see this as equivalent to disabling the Java bindings. Also, if we want the Go bindings then we'd need to add an explicit Go dependency.

Reply:
And, if I build the original EC on a node without Go then there is no issue.